torch/csrc/stable/version.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <torch/version.h>
```

Comment:
I do think it'd be good to move this file to torch/headeronly to maintain our invariant

Reply:
opened https://github.com/pytorch/pytorch/pull/164381